docs-website-src/content/blockers.md

```diff
@@ -47,29 +52,41 @@ You can discover many of these issues by downloading `elevate-cpanel` and runnin
 * **cPanel is up to date**
   * You will need to be on a version mentioned in the "Latest cPanel & WHM Builds (All Architectures)" section at http://httpupdate.cpanel.net/
   * Mitigation: `/usr/local/cpanel/scripts/upcp`
-* **nameserver**
-  * cPanel provides support for a myriad of nameservers. (MyDNS, nsd, bind, powerdns). On RHEL 8 based distros, it is preferred that you always be on PowerDNS.
+* **Name Server**
+  * cPanel provides support for a myriad of name servers. (MyDNS, NSD, BIND, PowerDNS). On RHEL 8 based distributions, it is preferred that you always be on PowerDNS.
   * Mitigation: `/scripts/setupnameserver powerdns`
 * **MySQL**
-  * We will upgrade you automatically to MariaDB 10.6 during elevation if you do not do this in advance.
+  * If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8 based distributions, it will need to be upgraded to a version that is. If the MySQL installation is managed by cPanel we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation. Declining the offer to upgrade MySQL will block the elevation. If the MySQL installation is managed by CloudLinux, then the upgrade must be performed manually.  This can be done by running the following command:
+  `/usr/local/cpanel/bin/whmapi1 start_background_mysql_upgrade version=10.6`
+  You will need to wait for the upgrade to be complete before re-running the elevate script. Elevation will block if a MySQL upgrade is in progress.
+  * The system **must** not be setup to use a remote database server.
 * Some **EA4 packages** are not supported on AlmaLinux 8.
   * Example: PHP versions 5.4 through 7.1 are available on CentOS 7 but not AlmaLinux 8. You would need to remove these packages before upgrading. Doing so might impact your system users. Proceed with caution.
-* The system **must** be able to control the boot process by changing the GRUB2 configuration.
-  * The reason for this is that the framework which performs the upgrade of distro-provided software needs to be able to run a custom early boot environment (initrd) in order to safely upgrade the distro.
+* The system **must** be able to control the boot process by changing the GRUB2 configuration (unless using the --no-leapp option).
```

Comment:
I'd rather we not mention --no-leapp. I'm ok documenting this elsewhere but it's really for script developers who might use it. It's going to confuse anyone else.

Reply:
More specifically, nobody should ever run --no-leapp by hand, it's for other code to invoke this so it doesn't seem right to document it here.

Reply:
I'll remove the --no-leapp references.